src/test/java/ru/training/at/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,98 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import ru.training.at.hw2.commonsteps.PrepostConditions;
+
+public class Exercise1Test extends PrepostConditions {
+
+    private SoftAssert softAssert = new SoftAssert();
```

Comment:
The SoftAssert is part of the test method, so should be converted to a local variable